README.md

```diff
@@ -76,18 +76,128 @@ Going to the location `/` on the server, you will be served with an HTML page th
 
 ![Interface](https://nitratine.net/posts/hit-counter/interface.png)
 
-## Documentation
-- [Server Setup](./docs/setup.md)
-    - [Local Setup](./docs/setup.md#local-setup)
-    - [PythonAnywhere Setup](./docs/setup.md#pythonanywhere-setup)
-- [Server Configuration](./docs/config.md)
-- [Usage](./docs/usage.md)
-    - [Getting an SVG](./docs/usage.md#getting-an-svg)
-    - [Getting the Count Raw](./docs/usage.md#getting-the-count-raw)
-    - [Getting a Count For a Site That Isn't Me](./docs/usage.md#getting-a-count-for-a-site-that-isnt-me)
-    - [Generating Links With A Tool Hosted By The Server](./docs/usage.md#generating-links-with-a-tool-hosted-by-the-server)
-- [Background](./docs/background.md)
-    - [Anti-Refresh Counting System](./docs/background.md#anti-refresh-counting-system)
+## Hosting Your Own Server
+- Clone the repo: `git clone https://github.com/brentvollebregt/hit-counter.git`
+- cd into the repo: `cd hit-counter`
+- Install requirements: `python -m pip install -r requirements.txt`
+- Run the server: `python server.py`
+
+I host this on [pythonanywhere.com](https://hitcounter.pythonanywhere.com/); to do this make sure you have cloned the repo into the filesystem and then create a new project. Modify the "WSGI configuration file" under the "Code" header in the "Web" tab. Change line 16 to import your script and restart the application using the green button at the top.
+
+```python
+from server import app as application
+```
+
+> If you want to enable HTTPS on pythonaywhere, set config.ENABLE_SSL in `config.py` to True.
+
+## Server Configuration
+- Enable SSL: Set an environment variable `ENABLE_SSL` to `true`.
+- Change Database location: Set an environment variable `DATABASE_FILE_PATH` to the file path of the database. 
+
+> Alternatively these config values can be manually set in `config.py`.
+
+### Domain whitelisting
+You can configure the server to only count hits to domains matching a certain pattern. To do so, add regular expression entries to `URL_WHITELIST_RE` in `config.py`, e.g. `r'github\.com'`.
+
+
+## Docker
```

Comment:
Could you put this in `/docs/docker.md`, add appropriate headings and link back in README.md please